src/pages/Checklist/index.tsx

```diff
@@ -499,6 +556,73 @@ export const Checklist = () => {
             </li>
           </ul>
         </section>
+        <section>
+          <Heading level={3}>Simulations</Heading>
+          <CheckBox
+            label={
+              <Text className="checkbox-label">
+                I have simulated how to recorver my keystore JSON files with
+                mnemonic.
+              </Text>
+            }
+          />
+          <ul className="sub-checklist-item">
+            <li className="py5">
+              <Text>
+                <strong>Hint:</strong> you can recover it with{' '}
+                <Link
+                  inline
+                  external
+                  to="https://github.com/ethereum/eth2.0-deposit-cli#existing-mnemonic-arguments"
+                >
+                  deposit-cli existing-mnemonic command
+                </Link>
+              </Text>
+              .
+            </li>
+          </ul>
+          <CheckBox
+            label={
+              <Text className="checkbox-label">
+                I have simulated power loss (server and internet) and automatic
+                resumption.
+              </Text>
+            }
+          />
+          <CheckBox
+            label={
+              <Text className="checkbox-label">
+                I have simulated how to migrate from one Eth2 client to another
+                Eth2 client.
+              </Text>
+            }
+          />
+          <CheckBox
+            label={
+              <Text className="checkbox-label">
+                I have simulated how to change my keystore passphrase and resume
```

Comment:
Is this something we expect users to know how to do? Surely it is easier to just delete and recreate your keystores?

Reply:
Right, I changed it to "I have simulated how to manually stop and restart my Beacon Node (BN) and Validator Client (VC) gracefully." now.

Reply:
Sounds good to me:)